### ISSUE TITLE
P1: Zero-shot pretrained forecaster (Chronos-2 / Toto-2 class) loaded by apr pull, with optional fine-tuning — CRUX-O-24

### ISSUE THREAD

**Issue by noahgift** [paiml/aprender#3394]
Parent: #3370 (CRUX category O — AutoML parity vs AutoGluon 1.6.3) · Tier **P1** · Milestone **0.70.0** · Contract `contracts/crux-O-24-v1.yaml` · demand 4/5 · intake `missing`

## The competitor verb

```python
hyperparameters={"Chronos2": {"fine_tune": True}}  # 1.5 spotlight; 1.6 adds Toto-2
```

Source: AutoGluon 1.6.3 checkout (`autogluon/VERSION`), surveyed 2026-09-16.

## The gap (measured in aprender at eb262f8eb)

apr pull fetches LLM checkpoints (Category A) and the inference crates run decoder transformers, but no forecasting head exists: nothing tokenises a numeric series into the model's input, samples a horizon, and maps it back to quantiles.

## Why it ranks where it does

Chronos-2 is the 1.5 spotlight and the default 'best' forecasting preset; it is the first foundation model in AutoGluon's forecasting tier and sits on the same inference substrate aprender already has.

## Acceptance criteria (each must be able to FAIL)

- [ ] **zero-shot output matches a pinned reference.** a pinned small checkpoint pulled by `apr pull` forecasts the committed fixture within 1e-3 of the reference quantiles. Binds to `zero_shot_matches_reference` in `forecast/pretrained/tests` of `aprender-core` when authored.
- [ ] **fine-tuning changes the weights and improves the in-domain metric.** after fine_tune on the fixture the WQL improves by >= 5% relative AND at least one weight tensor differs from the pulled checkpoint. Binds to `finetune_improves_and_changes_weights` in `forecast/pretrained/tests` of `aprender-core` when authored.
- [ ] The contract's falsification gates move from LIVE-PENDING to bound test citations, and `pv validate contracts/crux-O-24-v1.yaml` passes with `registry: false`.
- [ ] Mutation-verified: at least one gate above is shown RED under the named mutation before the PR merges.
- [ ] CRUX-SHIP-001: classifier/library code AND `apr` CLI wiring AND one e2e test land in the same PR, or a named blocker (BLOCKER-UPSTREAM-MISSING / BLOCKER-FIXTURE-ABSENT / BLOCKER-SPEC-OPEN) is recorded on the contract.

## Non-goals

Python/pandas API parity. The story is the WORKFLOW (what a user can ask for and what comes back), not the object model.

**Comment by noahgift** [comment]
triage 2026-09-23 (aprender-dd): milestoned to 0.76.0, theme Classical ML + AutoML. It is CRUX category O, AutoML parity vs AutoGluon, which no 0.70–0.75 theme covered. Operator selection 2026-09-23 ("New 0.76 release theme", wording by the cop aprender-cf); moved out of `backlog`.
